cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -923,6 +927,66 @@ public KafkaBrokerConfigurationBuilder withTieredStorage(String clusterName, Tie
         return this;
     }
 
+    /**
+     * Configures the quotas based on the type of the plugin - {@link QuotasPluginKafka}, {@link QuotasPluginStrimzi}
+     *
+     * @param clusterName    Name of the cluster
+     * @param quotasPlugin   Configuration of the quotas plugin
+     * @return  Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withQuotas(String clusterName, QuotasPlugin quotasPlugin) {
+        if (quotasPlugin != null) {
+            // for the built-in Kafka quotas plugin we don't need to configure anything
+            if (quotasPlugin instanceof QuotasPluginStrimzi quotasPluginStrimzi) {
+                printSectionHeader("Quotas configuration");
+
+                configureQuotasPluginStrimzi(clusterName, quotasPluginStrimzi);
+
+                writer.println();
```

Comment:
maybe it's me, but why the blank lines in between?

Reply:
it's same across the configurations -> it's more readable

Reply:
Fine if we already have in other places, but it's not blank lines making the code more readable imho.

Reply:
Ah you meant the lines between the code, I thought that you mean the `writer.println();`
I will remove the blank lines there, thanks (misread the comment, sorry)